front/src/layouts/footer/footer.css

```diff
@@ -0,0 +1,74 @@
+div.body {
+  color: white;
+  width: 1440px;
```

Comment:
variable

Reply:
1) it should be variable (1440px), it is like breakpoints
2) it is not about body's width.